core/utils/checksum.h

```diff
@@ -318,36 +280,28 @@ static inline uint16_t CalculateIpv4TcpChecksum(const Ipv4Header &iph,
 
 // Return incrementally updated internet checksum of old_checksum
 // when 'old_value' changes to 'new_value' e.g., changed IP address
-static inline uint16_t CalculateChecksumIncrementalUpdate(uint16_t old_checksum,
-                                                          uint32_t old_value,
-                                                          uint32_t new_value) {
+static inline uint16_t CalculateChecksumIncremental32(uint16_t old_checksum,
```

Comment:
Could you update the function comment (and the next function's comment) to distinguish them?